cfme/tests/cloud/test_provisioning.py

```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
         'first_name': 'Image',
         'last_name': 'Provisioner',
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
```

Comment:
Azure has a bit another naming/parameters, but these inside "not isinstance" are common for all other providers.Azure gets the same provisioning options inside "isinstance Azure"